doc/samples/variants/dut.rst

```diff
@@ -0,0 +1,46 @@
+.. _sidewalk_dut:
+
+Device Under Test
+#################
+
+This sample demonstrates a Device Under Test application.
+
+Overview
+********
+
+Device under test generic application having activated CLI including functionality for on_dev_certification for manufacturing process.
```

Comment:
I'm having troubles understanding this part. Can this be changed into a regular sentence?